src/App.tsx

```diff
@@ -1,99 +1,120 @@
-import React from 'react';
+import React, {
+  useState,
+  useRef,
+  useMemo,
+  useCallback,
+  useEffect,
+} from 'react';
+import { debounce } from 'lodash';
```

Comment:
Fix it
<img width="826" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/c8396766-d90a-4cc8-a726-e6131e3861c8">

Reply:
already installed??
![image](https://github.com/mate-academy/react_autocomplete/assets/103532899/37d23a46-d4cc-4d79-a1b3-37c8d2aba8bd)

Reply:
Still not fixed, you need to push your code after installed lodash
<img width="807" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/f11006e5-1417-45d6-8097-19f3e9aa82de">